alacritty.yml

```
@@ -455,18 +461,23 @@
   # If this is `true`, the cursor is temporarily hidden when typing.
   #hide_when_typing: false
 
-# Regex hints
+# Hints
 #
-# Terminal hints can be used to find text in the visible part of the terminal
-# and pipe it to other applications.
+# Terminal hints can be used to find text or hyperlinks in the visible part of
+# the terminal and pipe it to other applications.
 #hints:
   # Keys used for the hint labels.
   #alphabet: "jfkdls;ahgurieowpq"
 
   # List with all available hints
   #
-  # Each hint must have a `regex` and either an `action` or a `command` field.
-  # The fields `mouse`, `binding` and `post_processing` are optional.
+  # Each hint must have:
+  # 1. At least one of a `regex` or a `hyperlinks` field. It marks what content
+  #    this hint setting appies to. The former takes the matching regex,
+  #    and the latter takes `true` to match program marked hyperlinks.
```

```suggestion
  # 1. At least one of a `regex` or a `hyperlinks` field. It marks what content
  #    this hint setting applies to. Setting `hyperlinks` to `true` will match the hyperlink escape sequence.
```

Formatting isn't right, but I think this should be better. No need to point out that regex is a regex, the name kinda makes that obvious. Pointing out that this is about the content isn't particularly insightful either I think. So just keeping it short here.